src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -5,7 +5,7 @@ language_settings:
     interface_names:
       google.example.library.v1.LibraryService: Library
   python:
-    package_name: google.cloud.gapic.example.library.v1
+    package_name: google.cloud.library_v1.gapic
```

Comment:
What is the pattern for this? We should add the new pattern to the generated gapic config. Please create an issue to track.